components/schematics/ng-add/rules/rename-existing-imports.ts

```diff
@@ -0,0 +1,121 @@
+/**
+ * @license
+ * Copyright 2019 Dynatrace LLC
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import { Rule, SchematicContext } from '@angular-devkit/schematics';
+import { Tree } from '@angular-devkit/schematics/src/tree/interface';
+import { readFromTree } from '../ast';
+import * as ts from 'typescript';
+import { getImportModuleSpecifier } from '../ast/get-import-module-specifier';
+
+const OLD_IMPORT = '@dynatrace/angular-components';
+
+/** Rename every import in Filesystem to barista-components */
+export function renameExistingImports(files: string[]): Rule {
+  // tslint:disable-next-line: no-shadowed-variable
```

Comment:
I don't think we need this disable, there is no shadowed variable in the next line.